internal/cloud/realm/import_export_test.go

```diff
@@ -241,7 +241,7 @@ console.log('got heem!');
 			{
 				"name":   "yell",
 				"type":   "SCHEDULED",
-				"config": map[string]interface{}{"schedule": "0 0 * * 1"},
+				"config": map[string]interface{}{"schedule": "0 0 * * 1", "skip_catchup_events": false},
```

Comment:
The changes in this file are due to some existing test failures that I believe originated from [this commit](https://github.com/10gen/baas/commit/33f9daa54ec81dd3e36ac6780c413950604759be). Let me know if we should put this in a separate PR or commit message

Reply:
yeah I'm just noticing these failures too... turns out we added a field to the exported structure of db and scheduled triggers, so we'll need corresponding changes

we'll likely need this for both app data v1 _and_ v2, and i'm currently ironing out a few other failures related to v2 at the moment (looks related to data sources as I'm getting the `"unable to retrieve data source details"` error message back currently)

feel free to ignore this for now as I think I'll be able to merge #291 before this (it's super simple otherwise)